rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/DefaultImageNameFactory.groovy

```diff
@@ -75,10 +75,19 @@ public class DefaultImageNameFactory implements ImageNameFactory {
     // We need to replace the original fully-qualified package name with the unqualified package name before using
     // it in the target image name.
     def baseImagePackageName = osPackageName?.name ?: firstPackageName
-    def imageName = baseImagePackageName ? "$baseImagePackageName-" : ""
+
```

Comment:
Looks like line 77 (the `baseImagePackageName` declaration/assignment) and its comment can move inside of the `else` clause.